mail_debrand/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============
+Mail Debrand
+============
+
+This module modifies the functionality of emails to remove the Odoo branding.
+
+It also allows some context options to remove user and company signatures too.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Install it.
+* Send an email.
+* Nobody will know it comes from Odoo.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/205/10.0
+
+
+Known issues / Roadmap
+======================
+
+* This module relies on the translation of the strings here in this module that
+  must match the mail notification template translation on Odoo core,
+  specifically the words "using" and "Odoo".
```

Comment:
The solution is to ensure the many dozens of languages supported by Odoo are custom translated?
And what about base english? In the past there were problems with translations on it.